packages/client/components/NewMeeting.tsx

```diff
@@ -107,10 +107,20 @@ const useInnerWidth = () => {
   return innerWidth
 }
 
+const createMeetingOrder = ({standups}: {standups: boolean}) => {
+  const meetingOrder: [MeetingTypeEnum, ...MeetingTypeEnum[]] = ['poker', 'retrospective', 'action']
```

Comment:
Hey @BartoszJarocki, I'm just curious about the reason the deconstructing is used here? on the type of the `meetingOrder` array? How is it different from using `MeetingTypeEnum[]` directly?

Reply:
Hey @JimmyLv, I've asked the same question here: https://github.com/ParabolInc/parabol/pull/6082#discussion_r818787970
long story short, it specifies the type of an array with min 1 element. more info here: https://stackoverflow.com/questions/49910889/typescript-array-with-minimum-length

Reply:
oh yes! Thanks @BartoszJarocki I also marked it as #TIL of my notes, 😁
In the StackOverflow you just sent, I also found someone mentioned that extracting a Generic Types with better namely `type ArrayOneOrMore<T> = [T, ...T[]]`

https://stackoverflow.com/questions/62535983/typescript-array-with-arbitrary-minimum-length-n-which-can-be-large#comment110592890_62535983

Reply:
Does not work in all circumstances (like when an Array is required) but we already have this helper type: https://github.com/ParabolInc/parabol/blob/9e1b280bc2246d90426fa892b9be8d38e38ed881/packages/client/types/generics.ts#L89

Reply:
Make sense! @Dschoordsch Thanks to Georg for answering and sharing the context!